### PR TITLE
amp counts are now optional via env setting

### LINKED ISSUE #2: Multiple amps cause odd return when querying nonexistent
in my case I have two amps in series: master & slave 1

when querying a non-existent amp (>30\r) I receive a set of odd bytes on the receive line that are intermixed into the payload from the previous commands for amp one and amp two

I believe the fix is to environmentally set what your amp count is and react to that by not querying non existent amps.  Related, the error never happens on the first amp, ONLY the second amp.  I re-wired my amps by swapping their roles and the error maintained.  As soon as I stopped querying amp 3 it worked great.

Also, I removed amp number two and left the code unchanged and it worked great.  It only appears to be an issue on a two amp system when querying for three.

I'll submit a PR to deal with this.

@jnewland - **great** work on this; I love the encapsulation in node, makes it so easy to deal with on the PI

### PR DESCRIPTION
**Pull request by cbschuld**
if you have more than one amp you can now set that via the env setting AMPCOUNT

fixes #2

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -161,8 +162,8 @@ connection.on("open", function () {
   app.get('/zones/:zone/:attribute', function(req, res) {
     zones[req.zone] = undefined;
     connection.write("?10\r");
-    connection.write("?20\r");
-    connection.write("?30\r");
+    AmpCount >= 2 && connection.write("?20\r");
+    AmpCount >= 3 && connection.write("?30\r");
```
Comment (jnewland) [inline app.js]: Is it possible to have 4 amps in a chain, or is 3 the max?

**Comment by jnewland** [comment]
> This branch has conflicts that must be resolved

Can you rebase this against master or merge it and resolve the conflicts?

**Comment by cbschuld** [comment]
grrr; admin error on my end @jnewland - sorry - fixed!

File: app.js (hunk 1 above)
Comment (cbschuld) [reply app.js]: 3 is the max @jnewland

**Pull request closed without merging**